mbedtls/src/ssl/ciphersuites.rs

```diff
@@ -7,6 +7,7 @@
  * according to those terms. */
 
 use mbedtls_sys::types::raw_types::c_int;
+use mbedtls_sys::*;
```

Comment:
Why is this import here if nothing is being added to the file?

Reply:
I had to change `define_enum` macro to not assume the values are in `mbedtls_sys::` since we need to reference a value that is not defined in that crate (`CUSTOM_PK_TYPE`)

Reply:
I see